src/App.js

```diff
@@ -11,10 +13,10 @@ function App() {
   return (
     <div className="App">
       {pathname!=="/" && <Navbar/>}
+        <ToastContainer theme="light" autoClose={2000} position="top-right"/>
```

Comment:
Good that you've added toast in your app

Reply:
Thanks